request.lisp

```diff
@@ -127,7 +127,10 @@ body using the boundary BOUNDARY."
       (crlf))))
 
 (defun %read-body (stream element-type)
-  (declare (stream stream))
+  ;; On ABCL, a flexi-stream is not a normal stream. This is caused by
+  ;; a bug in ABCL which is supposedly quite difficult to fix. More
+  ;; details here: http://abcl.org/trac/ticket/377
+  #-abcl(declare (stream stream))
```

Comment:
Sure, it's done.

On 11 March 2015 at 13:23, Hans Hübner <notifications@github.com> wrote:

> In request.lisp
> <https://github.com/edicl/drakma/pull/52#discussion_r26189716>:
>
> > @@ -127,7 +127,10 @@ body using the boundary BOUNDARY."
> >        (crlf))))
> >
> >  (defun %read-body (stream element-type)
> > -  (declare (stream stream))
> > +  ;; On ABCL, a flexi-stream is not a normal stream. This is caused by
> > +  ;; a bug in ABCL which is supposedly quite difficult to fix. More
> > +  ;; details here: http://abcl.org/trac/ticket/377
> > +  #-abcl(declare (stream stream))
>
> I know it is a bit picky, but can you please put the #+abcl on a separate
> line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edicl/drakma/pull/52/files#r26189716>.
>